src/xcape/core.py

```diff
@@ -132,81 +138,83 @@ def calc_cape(*args, **kwargs):
+    ml_depth : float, optional (default is 500)
         Depth (m) of mixed layer. Only applies when the source='mixed-layer'
-    adiabat : {'pseudo-liquid', 'reversible-liquid','pseudo-ice', 'reversible-ice'}
-    pinc : float, optional
+    adiabat : str, optional, default is 'pseudo-liquid'
+        options include: 'pseudo-liquid', 'reversible-liquid','pseudo-ice', 'reversible-ice'  
+    pinc : float, optional, default is 500
         Pressure increment for integration (Pa) - Recommended usage (between 1000 and 100) is 
         based on desired speed, with accuracy of the calculation increasing with smaller 
         integration increments. 
-    method : {'fortran', 'numba'}
-        Option to select numerical approach using wrapped Fortran 90 or a Numba python variant.
-    vertical_lev : {'sigma', 'pressure'}
-        Option to select vertical grid, between model coordinates and pressure levels.
+    method : str, optional, default is 'fortran')
+        Option to select numerical approach using wrapped Fortran 90 or a Numba python variant (to be implemented)
+    vertical_lev : str, optional, default is 'sigma'
+        Option to select vertical grid, between model coordinates, 'sigma', and pressure levels, 'pressure'.
 
     Returns
     -------
-    cape : 'array-like'
+    cape : array-like
         Convective available potential energy (J/Kg)
-    cin : 'array-like'
+    cin : array-like
         Convective inhibition (J/Kg)
-    MUlev : 'array-like'
+    MUlev : array-like
         Most Unstable level location index (only returned for source: {'most-unstable'})
-    zMUlev : 'array-like'
+    zMUlev : array-like
         height of MUlev (m) (only returned for source: {'most-unstable'}
-     """
+        
+    Notes
+    -----
+    CAPE is calculated on a user specified set of parcel options based on the integration:
+    
+    
+    .. math:: CAPE = g \\int_{LFC}^{EL} (\Theta_{v,parcel} - \Theta_{v,env}/(\Theta_{v,env}) d\\text{dz}
+
+    .. math:: CIN = g \\int_{SFC}^{LFC} (\Theta_{v,parcel} - \Theta_{v,env})/(\Theta_{v,env}) d\\text{dz}
+    
+    * :math:`CAPE` = Convective available potential energy 
+    * :math:`CIN` = Convective inhibition
+    * :math:`LFC` = Level of free convection
+    * :math:`EL` = Equilibrium level
+    * :math:`g` = Gravitational acceleration
+    * :math:`\Theta_{v,parcel}` = Virtual potential temperature of the parcel
+    * :math:`\Theta_{v,env}` = Virtual potential temperature of the environment
+    * :math:`z` = height above ground    
+  
+    Examples
+    -------    
+    Example of usage:
+    
+    >>> cape, cin = core.calc_cape(p, t, td, ps, ts, tds, source ='mixed-layer',
+                    mldepth=500., adiabat='pseudo-liquid', pinc = 500., 
+                    method='fortran', vertical_lev='sigma')
+   
+    References
+    ----------
+    .. [1] prova
+
+    """
 
     if len(args)<6:
         raise ValueError("Too little arguments.")     
```

Comment:
Do we have to use the word little? Too few is preferable - but I don't know if this is typical for python.